### PR TITLE
Update admin-configuration.md

### PR DESCRIPTION
**Pull request by JamesDrummond**

### CODE REVIEW

File: src/main/_docs/admin-guide/admin-configuration.md [hunk 1]
```diff
@@ -166,7 +166,12 @@ If you'd like your users to work with projects which have their own Docker image
 
 These two tactics will allow user workspaces to perform `docker` commands from within their workspace to create and work with Docker containers that will be outside the workspace. In other words, this makes your user's workspace feel like their laptop where they would normally be performing `docker build` and `docker run` commands.
 
-You will need to make sure that your user's workspaces are powered from a stack that has a Docker client installed inside of it. Our defalt stacks do not have a Docker client installed, but we have sample stacks from Che-in-Che that have examples for how to handle this.
+You will need to make sure that your user's workspaces are powered from a stack that has Docker installed inside of it. Che's default images do not have Docker installed, but there is a sample docker image eclipse/alpine_jdk8 created from our [dockerfile](https://github.com/eclipse/che-dockerfiles/blob/master/recipes/alpine_jdk8/Dockerfile) that includes docker which can be used as new stack's base image. Refer to the [che-in-che tutorial]({{ base }}({{base}}{{site.links["tutorial-che-in-che"]}}) for additional information.
```
Comment (TylerJewell) [inline src/main/_docs/admin-guide/admin-configuration.md]: I find this confusing.  You start off talking about getting Docker in an image and then you jump to volume mounting the socket without an explanation for why that is necessary.

Comment (JamesDrummond) [inline src/main/_docs/admin-guide/admin-configuration.md]: Updated. Che to follow once this is approved/reviewed.

File: src/main/_docs/admin-guide/admin-configuration.md [hunk 2]
```diff
@@ -159,14 +159,43 @@ CODENVY_MACHINE_DOCKER_PRIVILEGE_MODE=true
...
-2. Configure Codenvy workspaces to volume mount their host Daemon when starting
+1. Activate Docker's privileged mode, where your user workspaces have access to the host.
+```shell
+# Update your codenvy.env to allow all Codenvy workspaces machines/containers privileged rights:
+CHE_DOCKER_PRIVILEGED=true;
+```
+2. Configure Codenvy workspaces to volume mount the host docker daemon socket file.
+```shell
+# Update your codenvy.env to allow all Codenvy workspaces to volume mount their host Daemon when starting:
+CODENVY_MACHINE_SERVER_EXTRA_VOLUME=/var/run/docker.sock:/var/run/docker.sock;
```
Comment (bmicklea) [inline src/main/_docs/admin-guide/admin-configuration.md]: Isn't this part of how Codenvy is always started? In other words isn't this the default behavior?

Comment (JamesDrummond) [inline src/main/_docs/admin-guide/admin-configuration.md]: This is only for using docker.sock inside a workspace. Yes for the codenvy server we always volume mount docker.sock into it but we do not volume mount into workspace machines the docker.sock file.

**Pull request merged**